<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewVersionDialog</class>
 <widget class="QDialog" name="NewVersionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SQLiteStudio updates</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="topLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>New updates are available!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="updateList">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
     <column>
      <property name="text">
       <string>Component</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Current version</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Update version</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="checkOnStartupWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="checkOnStartupCheck">
        <property name="text">
         <string>Check for updates on startup</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="updateButton">
     <property name="text">
      <string>Update to new version!</string>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/icons/img/get_update.png</normaloff>:/icons/img/get_update.png</iconset>
     </property>
     <property name="description">
      <string>The update will be automatically downloaded and installed. This will also restart application at the end.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="abortButton">
     <property name="text">
      <string>Not now.</string>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/icons/img/act_abort.png</normaloff>:/icons/img/act_abort.png</iconset>
     </property>
     <property name="description">
      <string>Skip the update and close this window.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
